webapp/channels/src/selectors/work_template.ts

```diff
@@ -9,10 +9,8 @@ import {GlobalState} from 'types/store';
 export const areWorkTemplatesEnabled = createSelector(
     'areWorktemplatesEnabled',
     (state: GlobalState) => getFeatureFlagValue(state, 'WorkTemplate') === 'true',
-    (state: GlobalState) => getFeatureFlagValue(state, 'BoardsProduct') === 'true',
-    (state: GlobalState) => Boolean(state.plugins.plugins?.playbooks),
-    (workTemplateFF, boardProductEnabled, pluginPlaybooksInstalled) => {
-        return workTemplateFF && boardProductEnabled && pluginPlaybooksInstalled;
```

Comment:
This playbooks condition is also temporarily because in mono-repo playbook is not running as plugin

Reply:
Same here, I believe we don't need these checks now in all running in product mode.

Reply:
👍🏻